sources/packages/web/src/components/aest/students/assessment/ManualReassessment.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <v-card>
```

Comment:
I believe that we created the `body-header-container` to implement similar layouts. The current page is just outdated but the below one can be used as a reference:
[src\components\common\OverawardDetails.vue](https://github.com/bcgov/SIMS/blob/955f54b299faaf9d377a7b6abf602e295fc83535/sources/packages/web/src/components/common/OverawardDetails.vue#L1-L9)